jerry-core/ecma/operations/ecma-typedarray-object.c

```diff
@@ -178,7 +168,7 @@ set_typedarray_element (lit_utf8_byte_t *dst, /**< the location in the internal
     }
     case LIT_MAGIC_STRING_UINT32_ARRAY_UL:
     {
-      SET_ELEMENT (uint32_t, dst, value);
+      *((uint32_t *) dst) = (uint32_t) value;
```

Comment:
Why uint32_t is a special case? If the value is negative float, the conversion is undefined http://stackoverflow.com/questions/2490600/iphone-floats-cast-to-unsigned-ints-get-set-to-0-if-they-are-negative